<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWidget</class>
 <widget class="QWidget" name="mainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>588</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>实验三 网络数据获取与显示</string>
  </property>
  <widget class="QtCharts::QChartView" name="chartview">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>588</width>
     <height>566</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>30</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>选择城市</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>290</y>
     <width>111</width>
     <height>84</height>
    </rect>
   </property>
   <property name="title">
    <string>查询选项</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="rbTemp">
      <property name="text">
       <string>查询气候</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbAQI">
      <property name="text">
       <string>查询AQI</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="savechart">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>410</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>130</y>
     <width>126</width>
     <height>145</height>
    </rect>
   </property>
   <property name="title">
    <string>选项</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCheckBox" name="cbShowPoint">
      <property name="text">
       <string>显示数据点</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cbLegendBold">
      <property name="text">
       <string>图注粗体</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cbLegendItalic">
      <property name="text">
       <string>图注斜体</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnLegendAlign">
      <property name="text">
       <string>图注(底)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="comboMonth">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>100</y>
     <width>87</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>80</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>选择时间</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboCity">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>50</y>
     <width>87</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QtCharts/QChartView&gt;
#include &lt;QtCharts/chartsnamespace.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
